index.html

```diff
@@ -93,7 +94,7 @@ <h2 class="stories">Multi-Post Stories</h2>
               <li>HTML</li>
               <li>JavaScript</li>
             </ul>
-            <button id="show" class="button">See Project</button>
+            <button id="show" class="button cbtn">See Project</button>
           </div>
         </div>
         <div class="recentworkscontainer">
```

Comment:
- [ ] Allow me to explain the concerns from the previous review. Quoted below are the project requirements that are in line with the requested change.

  > - In order to associate each project with the popup window details, you must use **a JavaScript array to store all of the information for all projects**.
  >   - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link the  to source.
  >   - You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
  >   - After that, you could implement the popup window.
  
  Let's go through them one by one.
  
  - The first requirement is that we `MUST use a JavaScript Array to store all the information for all projects`. What does that mean? It means they are no longer required in the HTML and should be refactored out into a JS file, i.e. `data.js`.
  - The second one describes how we're supposed to store the projects' information inside the array.
  - The third one is telling us to `update the main page so that the projects section is created dynamically using the information stored in that JavaScript object`. What does it mean to update? It means that we need to remove all projects' info that we had previously coded into the HTML and now load them dynamically with JavaScript.
  
  Please go through the project requirements again and consider working on them.